test/e2e/specs/fse/fse__limited-global-styles.ts

```diff
@@ -2,31 +2,45 @@
  * @group gutenberg
  */
 
-import { envVariables, TestAccount, FullSiteEditorPage } from '@automattic/calypso-e2e';
+import {
+	getTestAccountByFeature,
+	envVariables,
+	TestAccount,
+	FullSiteEditorPage,
+} from '@automattic/calypso-e2e';
 import { Browser, Page } from 'playwright';
 import { skipDescribeIf } from '../../jest-helpers';
 
 declare const browser: Browser;
 
 /**
- * Note: test user for this spec requires the use of FSE theme with
- * style variations (eg. Twenty Twenty-Three).
+ * Ensures the Full Site Editor's Limited Global Styles feature is functional.
  *
- * @see https://github.com/Automattic/wp-calypso/issues/78107
+ * Note: test user for this spec requires the use of FSE theme with style variations
+ * (eg. Twenty Twenty-Three).
  *
- * We skip Atomic sites because they are not affected by Limited Global Styles.
+ * @see https://github.com/Automattic/wp-calypso/issues/78107
  *
+ * Note: Atomic sites because they are not affected by Limited Global Styles.
```

Comment:
This part is a bit confusing now.
```suggestion
 * Note: We skip Atomic sites because they are not affected by Limited Global Styles.
```

Reply:
Good catch, thanks!